pulsar-client-reactive-jackson/src/test/java/org/apache/pulsar/reactive/client/jackson/PulsarReactiveClientModuleTests.java

```diff
@@ -507,6 +529,19 @@ void shouldSerDeserCryptoKeyReader() throws Exception {
 		assertThat(json).isEqualTo(expected);
 	}
 
+	@Test
+	void shouldSerDeserExponentialDeliveryBackoff() throws Exception {
```

Comment:
I think this should be named `shouldSerDeserRedeliveryBackoff`.

Reply:
Sure, fixed.